rerun_py/pyproject.toml

```
@@ -38,7 +38,7 @@ text = "MIT OR Apache-2.0"
 tests = ["pytest==8.4.1"]
 notebook = ["rerun-notebook==0.25.0-alpha.1+dev"]
 datafusion = ["datafusion==47.0.0"]
-all = ["notebook", "datafusion"]
+all = ["notebook", "datafusion==47.0.0"]
```

I think this `all` target was actually broken when I added it and has been silently ignored.

It looks like it should be:
`all = ["rerun-sdk[notebook]", "rerun-sdk[datafusion]"]` however that then hits the rerun-notebook bootstrapping issue our pixi config calls out so we probably need to remove the `[all]` in our pixi. If that's too big a detour here I can file a ticket (or just fix it) separately.

---

I totally agree, this `all` target is fubar. I tiny bit less so now, but I would much rather remove it entirely tbh. Out of scope here though.

---

Related: we probably should have the datafusion dep for the `notebook` extra.